src/data/posts/fountain-beta.md

```diff
@@ -0,0 +1,88 @@
+
+## Main focuses
+
+### Options
+Fountain is a new Yeoman generator allowing the user to choose the most structurant technologies you'll use:
+- Web framework
+
+  That's a key point of Fountain, you can choose either Angular 1, 2 or React and we'll provide us a starter app with this framework. And we mean a really adapted project with dev tools and test environment for the framework you chose.
+
+- Module and dependency management
+
+  This is certainly the second most structurant choice you have to make starting a Webapp these days. Webpack, SystemJS, any of them ? And believe us, this choice pull a lot of tooling and configurations to make. Fountain provide this simple choice and configure your project following it.
+
+- JavaScript version
+
+  It's became rare to start a webapp in pure browser compatible JavaScript. Everyone is about Babel or TypeScript but this needs tooling. Here you just have to choose.
+
+- Style (CSS) meta-language
+
+  The oldest preprocessors area. You just have to choose Sass or Less (or nothing).
+
+### Developer eXperience
+
+The tooling proposed is focused on offering the best developer experience in order that modern Web development benefits from the extremly dynamic ecosystem instead of creating fatigue.
+
+Whatever the options you choose, you'll get a development server embedded, live reload enabled, even hot module reload when possible.
+
+Of course, for delivering, you still have the optimization task to produce a state of the art optimized dist version.
+
+### Ready to go
+
+The main point of Fountain is to deliver a stable project whatever the very different options that you could choose and always keep up to date with latest versions of every frameworks and tools.
+
+### Composition
+
+From the user point of view, Fountain is a very complete generator. From the inside, it's a complete ecosystem of various and specialized generators. This architecture will allow easier development and extension.
+
+## For the gulp-angular users
+
+We always seen Fountain as the gulp-angular 2.0. As Angular 1 and Bower was less and less popular we wanted not to remove them but to make an option of them.
+
+But asking if you want one router on another in an Angular app is not as difficult as completly switching frameworks. So, we needed a better architecture and that made us decide for a reboot project instead of an evolution.
+
+At the point of choosing for a reboot project, we had the freedom of redefining our major goals. Some are the same as before like providing clean projects, following best practicies and good test coverage. Others changed as we focused on structurant options in place of many smaller ones. The projects offered are also smaller as we discovered that our users was often more scared than helped receiving a big example project.
```

Comment:
- The generated projects ?
- our users were often scared than helped when they receive